lib/Reader/abisignature.cpp

```diff
@@ -312,7 +312,7 @@ CallSite ABICallSignature::emitUnmanagedCall(GenIR &Reader, Value *Target,
 
 Value *ABICallSignature::emitCall(GenIR &Reader, Value *Target, bool MayThrow,
                                   ArrayRef<Value *> Args,
-                                  Value *IndirectionCell,
+                                  Value *IndirectionCell, bool IsJmp,
```

Comment:
jmp is different from tail.call. For example, tail.call doesn't have to have the same signature as the caller so we don't have to pass a secret parameter. I would rather keep this as IsJmp for now to make it clear that this code has only been implemented and tested for jmp. When we get to tail calls we will probably do something like what you suggested.